homeassistant/components/climate/__init__.py

```diff
@@ -325,13 +287,13 @@ def target_temperature_low(self):
         return None
```

Comment:
I wonder if we should collapse the temperatures? Have the entity return either a single value or a list? Now theoretically it could return `None`

Reply:
It's how it works with a supported feature. But I agree, most of the entity are focus to use it on UI instead what is the best way to handle it internally

Reply:
What do you prefer? I think for influxdb and other things like that is a flat entity not bad like we need on UI